src/org/labkey/test/tests/visualization/ColumnChartTest.java

```diff
@@ -86,42 +87,41 @@ private void doSetup()
 
         log("Set the '" + PREGNANCY_COLUMN_NAME + "', '" + LANGUAGE_COLUMN_NAME + "', and '" + SIGNATURE_COLUMN_NAME + "' fields to be dimensions but not measures.");
 
-        waitForText("Dataset Fields");
-        PropertiesEditor editor = PropertiesEditor.PropertiesEditor(getDriver()).withTitleContaining("Dataset Fields").find();
-        editor.selectField(PREGNANCY_COLUMN_NAME);
-        PropertiesEditor.FieldPropertyDock.ReportingTabPane pane = editor.fieldProperties().selectReportingTab();
-        pane.setDimension(true);
-        pane.setMeasure(false);
-        editor.fieldProperties().selectReportingTab().setDimension(true);
+        DatasetDesignerPage datasetDesignerPage = new DatasetDesignerPage(getDriver());
+
+        DomainFormPanel domainFormPanel = datasetDesignerPage.getFieldsPanel();
+        domainFormPanel.getField(PREGNANCY_COLUMN_NAME)
+                .setDimension(true)
+                .setMeasure(false);
         DATA_SOURCE_1_DIMENSIONS.add(PREGNANCY_COLUMN_NAME);
 
-        editor.selectField(LANGUAGE_COLUMN_NAME);
-        pane.setDimension(true);
-        pane.setMeasure(false);
+        domainFormPanel.getField(LANGUAGE_COLUMN_NAME)
+            .setMeasure(true)
```

Comment:
@labkey-chrisj this should be `setDimension(true)`

Reply:
lol whoops